apps/api/src/app/events/services/push-service/handlers/fcm.handler.ts

```diff
@@ -11,13 +11,14 @@ export class FCMHandler extends BasePushHandler {
   buildProvider(credentials: ICredentials) {
     const credentialConfig: IFcmConfig = { user: credentials.user, serviceAccount: credentials.serviceAccount };
 
-    if (!credentials.user && !credentials.serviceAccount) {
-      throw new Error('Config is not valid for fcm');
-    }
```

Comment:
What's the reason behind removing this check? I do not find it harmful.

Reply:
this code was not removed but the if condition changed a little, but basically does the same

Reply:
My bad. Apologies for the mistake.